org.hl7.fhir.r5/src/main/java/org/hl7/fhir/r5/renderers/utils/RenderingContext.java

```diff
@@ -176,6 +209,12 @@ public boolean isXml() {
     }
   }
 
+  public enum MultiLanguagePolicy {
+    NONE, // ONLY render the language in the locale

```

Comment:
The only usage of `MultiLanguagePolicy.NONE`, seems to deviate from the comment: `// ONLY render the language in the locale`. It looks to me like in CodeSystemRenderer, it's rendering according to the default definition instead of requesting any locale.

Reply:
that's fixed in my next commit